app/javascript/react/screens/App/Settings/Settings.js

```diff
@@ -1,82 +1,84 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { reduxForm, Field } from 'redux-form';
-import { Breadcrumb, Form, Button, Spinner } from 'patternfly-react';
+import { Breadcrumb, Tabs, Tab } from 'patternfly-react';
 import Toolbar from '../../../config/Toolbar';
-import NumberInput from '../common/forms/NumberInput';
+import GeneralSettings from './GeneralSettings';
+import ConversionHostsSettings from './ConversionHostsSettings';
 
-export class Settings extends React.Component {
-  componentDidMount() {
-    const { fetchServersAction, fetchServersUrl, fetchSettingsAction, fetchSettingsUrl } = this.props;
-    fetchServersAction(fetchServersUrl);
-    fetchSettingsAction(fetchSettingsUrl);
-  }
+const Settings = props => {
+  const {
+    match,
+    redirectTo,
+    fetchServersAction,
+    fetchSettingsAction,
+    patchSettingsAction,
+    isFetchingServers,
+    isFetchingSettings,
+    isSavingSettings,
+    servers,
+    savedSettings,
+    settingsForm
+  } = props;
 
-  onApplyClick = () => {
-    const { patchSettingsAction, servers, settingsForm } = this.props;
-    patchSettingsAction(servers, settingsForm.values);
-  };
+  const toolbarContent = (
+    <Toolbar>
+      <Breadcrumb.Item href="/dashboard/maintab?tab=compute">{__('Compute')}</Breadcrumb.Item>
+      <Breadcrumb.Item href="#/plans">{__('Migration')}</Breadcrumb.Item>
+      <Breadcrumb.Item active>{__('Migration Settings')}</Breadcrumb.Item>
+    </Toolbar>
+  );
 
-  render() {
-    const { isFetchingServers, isFetchingSettings, isSavingSettings, savedSettings, settingsForm } = this.props;
+  const generalSettings = (
+    <GeneralSettings
```

Comment:
Wondering if it would make sense, architecturally, to create a container component for GeneralSettings instead of passing these props down from Settings.

Also ok with putting this off until the actual need arises, but just want to avoid another <Overview /> situation, lol.

Do we anticipate any of these props to be shared across tabs?

Reply:
That's a good point, I guess I wasn't sure if we would need to use the generalized /api/settings actions and state outside of that first tab. I think you're right, I'll connect each tab to redux separately.